mne/decoding/ssd.py

```diff
@@ -125,9 +124,9 @@ def __init__(
                 "The signal band-pass must be within the noise "
                 "band-pass!"
             )
-        self.picks_ = _picks_to_idx(info, picks, none="data", exclude="bads")
+        self.picks = _picks_to_idx(info, picks, none="data", exclude="bads")
```

Comment:
there was a reason for this. Technically you should be able to do:

est = SSD(info, ..., picks=picks1)
est.fit(...)
est.picks = picks2
est.fit(...)

and it should just work. Now I see that it's already broken. The gist is that you should be able to overwrite init param and do a fit with the same instance.

so the init should just do self.picks = picks
and self.picks_ should be set in the fit.

Reply:
Hah, I think more than one of our estimators do not work this way. But I can probably at least fix it here. Will push a fix